laterpay/application/Controller/Base.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * LaterPay base controller.

```

Comment:
This makes me ask, what the base controller does. Just stating the name is not exactly helpful.